suites/test_module_edit_publish.py

```diff
@@ -83,7 +83,7 @@ def publish_module(self, api_auth):
     response = api_auth.get(self.request_url)
     # Check that the node has been marked as released
     check_that("The published module now has a 'released' node", response.json()["en_US"]["variants"][self.variant],
-               contains_string("released"))
+               has_entry("released"))
```

Comment:
@noopurmath3 what was your finding that "contains string" was failing but "has_entry" would make it pass?

Reply:
@anarang yes its working with has_entry() and I think has_entry() is better suited here than contains_string()